src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
<img width="1039" alt="Screenshot 2022-01-17 at 17 26 21" src="https://user-images.githubusercontent.com/33026471/150087511-97f96a58-3277-4e25-aa84-e7471efe6bca.png">

Reply:
fixed declaration in the latest commit